src/emoji/Emoji.js

```diff
@@ -1,6 +1,34 @@
-import { createIconSet } from 'react-native-vector-icons';
-import emojiMap from './emojiMap';
+import {createIconSet} from 'react-native-vector-icons';
+import emojiMap, { emoticonToEmojiMap, commonEmoticonList, textCodeList, emojiToTextCodeMap } from './emojiMap';
 
 const Icon = createIconSet(emojiMap, 'AppleColorEmoji');
 
 export default Icon;
+
+const getEmojiFromEmoticons = (emoticons: string): string => String.fromCodePoint(parseInt(emoticons, 16));
```

Comment:
You better move these functions to another file. This file will be solely for 'Emoji Component' and not everything about emojis (despite the name).